salt/modules/win_path.py

```diff
@@ -68,9 +78,12 @@ def get_path():
 
         salt '*' win_path.get_path
     '''
-    ret = __salt__['reg.read_value']('HKEY_LOCAL_MACHINE',
-                                   'SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Environment',
-                                   'PATH')['vdata'].split(';')
+    ret = salt.utils.stringutils.to_unicode(
```

Comment:
__salt__['reg.*'] always returns Unicode since 2016.11, if it does not its a bug.